src/index.ts

```diff
@@ -313,67 +363,215 @@ async function wizardCreateStatistics(cashlinks, folder) {
     }
+    console.log('Cashlink theme changed.\n');
+    return true;
+}
 
-    console.log('\nAll operations finished :)');
-    if (operations.includes(Operation.FUND) || operations.includes(Operation.CLAIM)) {
-        console.log(
-            'Transactions might still be pending in your local node and waiting to be relayed to other network nodes.\n'
-            + 'Make sure to check your wallet balance and keep your node running if needed.',
-        );
+/**
+ * Main application entry point
+ * Handles the complete workflow of cashlink operations
+ * including creation, modification, funding, and claiming
+ */
+async function main() {
+    try {
+        const config = getConfig();
+        const client = new RpcClient(config.nodeIp, config.nodePort);
+        console.log('Welcome to the cashlink generator!\n');
+        
+        // Initialize variables
+        let cashlinks: Map<string, Cashlink>;
+        let shortLinks: Map<string, string> | null;
+        let imageFiles: ImageFiles = new Map();
+        let folder: string;
+        let operations: OperationType[];
+        let shouldExport = false;
+
+        // Handle import or create new cashlinks
+        const importResult = await wizardImportCashlinks();
+        if (importResult) {
+            ({ cashlinks, shortLinks, imageFiles, folder } = importResult);
+            const operation = await prompt(
+                `What do you want to do? [${Object.values(Operation).join('/')}]: `
+            );
+            if (!Object.values(Operation).includes(operation as OperationType)) {
+                throw new Error(`Unsupported operation ${operation}`);
+            }
+            operations = [operation as OperationType];
+        } else {
+            const createResult = await wizardCreateCashlinks();
+            cashlinks = createResult.cashlinks;
+            shortLinks = createResult.shortLinks;
+            folder = createFolder();
+            operations = [Operation.CREATE_IMAGES, Operation.FUND];
+            shouldExport = true;
+        }
+
+        // Process operations
+        if (operations.includes(Operation.CREATE_IMAGES)) {
+            const oldImageFiles = imageFiles;
+            imageFiles = await wizardCreateImages(cashlinks, shortLinks, folder);
+            shouldExport = shouldExport || !oldImageFiles.size
+                || [...oldImageFiles.values()][0] !== [...imageFiles.values()][0];
+        }
+
+        if (operations.includes(Operation.CHANGE_MESSAGE)) {
+            shouldExport = shouldExport || await wizardChangeMessage(cashlinks);
+        }
+
+        if (operations.includes(Operation.CHANGE_THEME)) {
+            shouldExport = shouldExport || await wizardChangeTheme(cashlinks);
+        }
+
+        if (shouldExport) {
+            console.log('Exporting cashlinks.');
+            const file = `${folder || '.'}/cashlinks${importResult ? ` (update ${getCurrentDateString()} ${operations.join(' ')})` : ''}.csv`;
```

Comment:
The previous
```suggestion
            const file = `${folder || '.'}/cashlinks`
                + (importResult ? ` (update ${getCurrentDateString()} ${operations.join(' ')})` : '')
                + '.csv';
```
was more readable.